javascript/intermediate/multiple-cash-registers_sylvain1811.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Hacktoberithms - Multiple Cash Registers
+ * https://github.com/the-vampiire/hacktoberithms/issues/68
+ * By sylvain1811, 7 oct 2019
+ * node version v12.9.1
+ */
+
+function checkout_time(customers, n_cashier) {
+    let cashiers = Array(n_cashier).fill(0)
+    customers.forEach(customer => {
+        minIndex = cashiers.reduce((accIdx, current, index) => {
+            return current < cashiers[accIdx] ? index : accIdx
+        }, 0)
+        cashiers[minIndex] = cashiers[minIndex] + parseInt(customer)
+    });
+    return Math.max(...cashiers)
+}
+
+function test() {
```

Comment:
nice work building in your own test runner! you might be interested in the node `assert` module that comes in the standard lib. [you can read more about it here](https://nodejs.org/api/assert.html)

Reply:
I will take a look, thanks!